manager/src/main/java/org/openremote/manager/setup/builtin/ManagerDemoSetup.java

```diff
@@ -74,6 +74,7 @@
     public static GeoJSONPoint AREA_3_LOCATION = new GeoJSONPoint(5.487478, 51.446979);
     public static final String agentProtocolConfigName = "simulator123";
     public static final String thingLightToggleAttributeName = "light1Toggle";
+
```

Comment:
Please revert as it was before

Reply:
What exactly needs to be reverted here? I think this was commited in the latest version of the openremote:master, since this change happened when I pulled the latest version into our fork.

Reply:
Your file should match what is in master I shouldn't see any changes on this file compared to master.